src/ethereum/mod.rs

```diff
@@ -105,6 +109,10 @@ pub struct Options {
     #[clap(long, env, default_value = "1000")]
     pub max_log_blocks: usize,
```

Comment:
Is this still being used?

Reply:
https://github.com/worldcoin/signup-sequencer/pull/255#discussion_r1020286663 - it's still used to configure `page_size`